### PR TITLE
Cleanup & TC added l2_norm graph transformation.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Clean-up: replace FindOperator with FindOp and add missing unit tests
for the l2_norm graph transformation.

### CODE REVIEW

File: tensorflow/lite/toco/graph_transformations/identify_l2_normalization.cc [hunk 1]
```diff
@@ -150,7 +136,7 @@ ::tensorflow::Status IdentifyL2Normalization::Run(Model* model,
   AddMessageF("Creating %s replacing equivalent subgraph", LogName(*l2norm_op));
 
   // Erase the subgraph that is now replaced by L2Normalization
-  model->operators.erase(FindOperator(model, square_op));
+  model->operators.erase(FindOp((*model), square_op));
```
Comment (multiverse-tf) [inline tensorflow/lite/toco/graph_transformations/identify_l2_normalization.cc]: Just a nit: "*model" is fine to me.

**Comment by multiverse-tf** [comment]
Change the commit description to be more specific? Like "Clean-up: replace FindOperator with FindOp and add missing unit tests for the l2_norm graph transformation"?

File: tensorflow/lite/toco/graph_transformations/identify_l2_normalization.cc (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/toco/graph_transformations/identify_l2_normalization.cc]: @multiverse-tf , thanks for the review i have updated as per your suggestion, kindly check.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@multiverse-tf , thanks for the review i have updated as per your suggestion, kindly check.

Regards
Amit

**Review by multiverse-tf — APPROVED** [review]

**Comment by amitsrivastava78** [comment]
@multiverse-tf , thanks for approving the PR.
@gbaned , @rthadur please change the PR label to approved.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@miaout17 , can you please review the PR and provide your feedback.

Regards
Amit

**Pull request merged**